moveit_ros/visualization/motion_planning_rviz_plugin/src/ui/motion_planning_rviz_plugin_frame.ui

```diff
@@ -454,130 +584,31 @@
          <item>
           <widget class="QGroupBox" name="groupBox_12">
            <property name="title">
-            <string>Workspace</string>
+            <string>Path Constraints:</string>
```

Comment:
The colon needs to be removed maybe? And the margin to the select box is somewhat small.